Ways to train content changes

EmmaFrith opened the pull request:
Remove so many distractions/ call to actions from the intro, so that the page stays focused on 'ways to train'. 

Light copy edit throughout. 

Next steps: consider deleting the link to the long guidance page and bring any relevant info forward to this page instead.

EmmaFrith committed:
copy edit to intro

EmmaFrith committed:
copy edit to intro

EmmaFrith committed:
typo

EmmaFrith committed:
typo

EmmaFrith committed:
Update ways-to-train.md

EmmaFrith committed:
Update ways-to-train.md

peteryates commented on content/ways-to-train.md:
```diff
@@ -110,104 +120,128 @@ calls_to_action:
+You need to get Qualified Teacher Status (QTS) to be a primary or secondary teacher in a state school in England.
 
-### Things to consider
+You can get QTS in different ways.
```
Might it be worth listing those ways here and linking to the relevant section below?

EmmaFrith replied:
that's a good idea, although I'm about to do a more structural review of this page so maybe we can park that one?

peteryates commented on content/ways-to-train.md:
```diff
@@ -110,104 +120,128 @@ calls_to_action:
-You’ll need (or be about to finish) a doctorate in the [subject you want to teach](/guidance/become-a-teacher-in-england#researchers-in-schools-candidates-with-a-doctorate). You’ll then train over 3 years with a bursary. If you have a lot of relevant work experience you could [train and earn a salary](https://thebrilliantclub.org/researchers-in-schools/ris-applicants/ris-training-routes/salary-ro ...
-
-### Postgraduate Early Years Initial Teacher Training (EYITT)
+To go  down the [researchers in schools](/guidance/become-a-teacher-in-england#researchers-in-schools-candidates-with-a-doctorate) route you’ll need a doctorate, or be about to receive one, in the subject you want to teach.
```
```suggestion
To go down the [researchers in schools](/guidance/become-a-teacher-in-england#researchers-in-schools-candidates-with-a-doctorate) route you’ll need a doctorate, or be about to receive one, in the subject you want to teach.
```

peteryates reviewed (commented):
This looks like a big improvement and the language feels clearer and more concise.

EmmaFrith committed:
Update content/ways-to-train.md

Co-authored-by: Peter Yates <peter.yates@graphia.co.uk>

MylesJarvis reviewed (dismissed):
This is a huge improvement without losing any policy substance.

MylesJarvis committed:
Remove undergrad option 

This is so we can review with policy, in line with content process, but keep light copy edit which doesn't change anything

MylesJarvis committed:
Remove beta- URL

peteryates commented on content/ways-to-train.md:
```diff
@@ -134,80 +132,94 @@ PGCE programmes combine academic study with a minimum of 24 weeks spent training
 
-School Direct (fee-funded) programmes mean you’ll learn on the job while you work towards your Qualified Teacher Status (QTS). You’ll be based in a school, but you won’t be employed by them. You’ll learn on the job and be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
+You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+
+You’ll be based in a school but you will not be employed by them.
 
-Most School Direct programmes lead to a PGCE, but not all do — check with the training provider before applying.
+You’ll also be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
```
There are three consecutive paragraphs beginning with the word `You'll` here, and several others below. Perhaps it could be mixed up a bit?

peteryates reviewed (dismissed):
Look good overall. Left one comment about starting lots of paragraphs with `You'll`, but either way it's still a big improvement.

EmmaFrith committed:
Merge branch 'ways-to-train-content-changes' of https://github.com/DFE-Digital/get-into-teaching-content into ways-to-train-content-changes

peteryates committed:
Re-remove the undergraduate copy

MylesJarvis committed:
re-remove the steps changes
